src/controllers/refreshIssueTable.gs

```diff
@@ -0,0 +1,156 @@
+/**
+ * @file Contains controller class and dialog/callback method for inserting issue tables from jira filter.
+ */
+
+/**
+ * @desc Wrapper: Sidebar for "Refresh IssueTable"
+ */
+function menuRefreshIssueTable() {
+  RefreshIssueTable_Controller_.sidebar();
+}
+
+/**
+ * @desc Wrapper: Sidebar callback handler for initialization of sidebar content
+ * @return {object} Object({status: [boolean], tables: [object]})
+ */
+function cbRefreshIssueTable_initSidebar() {
```

Comment:
Just a personal opinion - but I find it more readable to say callbackRefreshIssueTable then cbRefreshIssueTable
- I have to think every time what "cb" means.

Reply:
hmmm yeah...    i prefer shortcuts and less letters :)  but i can adjust to your style. Makes sense.